config/rbac_default/kustomization.yaml

```diff
@@ -0,0 +1,38 @@
+namePrefix: eksa-
+
+bases:
+- ../rbac
+
+configurations:
+- ./apply-namespace.yaml
+
+transformers:
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: notImportantHere2
+      namespace: eksa-system
+    unsetOnly: true
+
+patchesJson6902:
+- patch: |-
+    - op: replace
+      path: /metadata/name
+      value: eksa-packages-rolebinding
+  target:
+    group: rbac.authorization.k8s.io
+    kind: RoleBinding
+    name: eksa-manager-rolebinding
+    namespace: eksa-packages
+    version: v1
+- patch: |-
+    - op: replace
+      path: /metadata/name
+      value: eksa-packages-role
+  target:
+    group: rbac.authorization.k8s.io
+    kind: Role
+    name: eksa-manager-role
+    namespace: eksa-packages
+    version: v1
```

Comment:
Does this mean that every time we add a role for a different namespace we would need to add a couple entries here?

Reply:
Yes, we would need to patch the name of the role for other namespaces since they never get created if there is already another role with the same name in eksa-system.